SYCL/Plugin/level_zero_dynamic_batch_test.cpp

```diff
@@ -166,7 +166,7 @@ int main(int argc, char *argv[]) {
   validate(Y1, Z7, M * N);
   validate(Y1, Z8, M * N);
 
-  for (size_t i = 0; i < 20; i++) {
+  for (size_t i = 0; i < 25; i++) {
```

Comment:
Could you please clarify why this change is needed. Does test fail when 20 iterations? Or do you want to test some specific behaviour with 25 iterations?